api/user/auth.service.ts

```diff
@@ -34,6 +39,30 @@ export class AuthService {
     }
   }
 
+  async resetPassword(email: string): Promise<boolean> {
+    const user = await this.getUserByEmail(email)
+    if (!user) {
+      return true
+    }
+    const token = generateToken()
+    const hashedToken = await this.hashString(token)
+    this.redisClient.set(
+      `forgot-password-${email}`,
+      hashedToken,
+      'ex',
+      1000 * 60 * 60 * 24
```

Comment:
Isn't EX seconds?

Reply:
Right 😅

Reply:
Sorry, I miscommunicated. I meant isn't this 1 000 days?

Reply:
Oh!, It is in milliseconds.

Reply:
Also, that 'ex' should be in caps.
Thanks for pointing it out.